script.js

```diff
@@ -1,50 +1,71 @@
-function Animal(name){
+
+class Animal {
+  constructor(name) {
     this.name = name;
-}
+  }
 
-Animal.prototype.action = function(){
-  document.getElementById(this.actionSoundName).play();
+  getImage() {
+    document.getElementById("animalImage").src = this.url;
+
+  }
   
+action() {
+  document.getElementById(this.actionSoundName).play();
+
 }
 
-Animal.prototype.putInTheDocument = function(){
-    var petsTable = document.getElementById("petsTable");
-    var petTR = document.createElement("tr");
-  
-    var petNameTD = document.createElement("td");
-    petNameTD.textContent = this.name;
-    petTR.appendChild(petNameTD);
-  
-    var petLegsTD = document.createElement("td");
-    petLegsTD.textContent = this.legs;
-    petTR.appendChild(petLegsTD);
-  
-    var petActionTD = document.createElement("td");
-    var petActionTDButton = document.createElement("button");
-    petActionTDButton.textContent = this.actionText;
-    petActionTD.appendChild(petActionTDButton);
-    petTR.appendChild(petActionTD);
-    
-    petActionTDButton.onclick = this.action.bind(this);
-    petsTable.querySelector("tbody").appendChild(petTR)
+putInTheDocument () {
+  var petsTable = document.getElementById("petsTable");
+  var petTR = document.createElement("tr");
+
+  var petNameTD = document.createElement("td");
+  petNameTD.textContent = this.name;
+  petTR.appendChild(petNameTD);
+
+  var petLegsTD = document.createElement("td");
+  petLegsTD.textContent = this.legs;
+  petTR.appendChild(petLegsTD);
+
+  var petActionTD = document.createElement("td");
+  var petActionTDButton = document.createElement("button");
+  petActionTDButton.textContent = this.actionText;
+  petActionTD.appendChild(petActionTDButton);
+  petTR.appendChild(petActionTD);
+
+  petActionTDButton.onclick = this.action.bind(this);
+  petNameTD.onclick = this.getImage.bind(this);
```

Comment:
td'ye değil tüm satıra yani tr'ye onclick ataman lazım. ` petTR.onclick = this.getImage.bind(this);`